extensions/aissemble-extensions-model-training-api-sagemaker/pyproject.toml

```diff
@@ -16,7 +16,7 @@ pydantic = ">=1.8.0,<2.0.0"
 kubernetes = ">=26.1.0"
 urllib3 = "^1.26.18"
 krausening = ">=19"
```

Comment:
S (Substantive): just to be sure, do we know why this was capped before and if that is a potential concern w/ this update?

Reply:
Strike that - missed your comment in the PR!

Reply:
The prior PR to make this change mentioned it was for Python 3.11.4 compatibility.  However, it seems 2.218 _is_ compatible (could have been a temporary issue that was resolved) and the [PyPI page](https://pypi.org/project/sagemaker/) lists 3.11 in the supported versions for the latest release.  Locally, I also see that Poetry is able to resolve this version with Python 3.11.4 just fine.
![Screenshot 2024-05-07 at 10 08 37 AM](https://github.com/boozallen/aissemble/assets/80470879/279d7158-9724-4319-b4c3-80e15b64f0fa)